docs/running/wdl.rst

```diff
@@ -3,15 +3,121 @@
-the needed json input file.  Full documentation can be found on the repository_, and a precompiled jar binary can be
+Running WDL with Toil
+---------------------
+
+You can run WDL workflows with ``toil-wdl-runner``. Currently,
+``toil-wdl-runner`` works by using MiniWDL_ to parse and interpret the WDL
+workflow, and has support for workflows in WDL 1.0 or later (which are required
+to declare a ``version`` and to use ``inputs`` and ``outputs`` sections).
+
+You can write workflows like this by following the `official WDL tutorials`_.
+
+When you reach the point of `executing your workflow`_, instead of running with
+Cromwell:
+
+``java -jar Cromwell.jar run myWorkflow.wdl --inputs myWorkflow_inputs.json``
+
+you can instead run with ``toil-wdl-runner``:
+
+``toil-wdl-runner myWorkflow.wdl --inputs myWorkflow_inputs.json``
+
+.. _`official WDL tutorials`: https://wdl-docs.readthedocs.io/en/stable/
+.. _`executing your workflow`: https://wdl-docs.readthedocs.io/en/stable/WDL/execute/
+
+This will default to executing on the current machine, with a job store in an
+automatically determined temporary location, but you can add a few Toil options
+to use other Toil-supported batch systems, such as Kubernetes:
+
+``toil-wdl-runner --jobStore aws:us-west-2:wdl-job-store --batchSystem kubernetes myWorkflow.wdl --inputs myWorkflow_inputs.json``
+
+For Toil, the ``--inputs`` is optiona, and inputs can be passed as a positional
+argument:
+
+``toil-wdl-runner myWorkflow.wdl myWorkflow_inputs.json``
+
+You can also run workflows from URLs. For example, to run the MiniWDL self test
+workflow, you can do:
+
+``toil-wdl-runner https://raw.githubusercontent.com/DataBiosphere/toil/36b54c45e8554ded5093bcdd03edb2f6b0d93887/src/toil/test/wdl/miniwdl_self_test/self_test.wdl https://raw.githubusercontent.com/DataBiosphere/toil/36b54c45e8554ded5093bcdd03edb2f6b0d93887/src/toil/test/wdl/miniwdl_self_test/inputs.json``
+
+Toil WDL Runner Options
+-----------------------
+
+**'-\\-jobStore'**: Specifies where to keep the Toil state information while
+running the workflow. Must be accessible from all machines.
+
+**'-o'** or **'-\\-outputDirectory'**: Specifies the output folder to save
+workflow output files in. Defaults to a new directory in the current directory.
+
+**'-m'** or **'-\\-outputFile'**: Specifies a JSON file to save workflow output
+values to. Defaults to standard output.
+
+**'-i'** or **'-\\-input'**: Alternative to the positional argument for the
+input JSON file, for compatibility with other WDL runners.
+
+**'-\\-outputDialect'**: Specifies an output format dialect. Can be
+``cromwell`` to just return the workflow's output values as JSON or ``miniwdl``
+to nest that under an ``outputs`` key and includes a ``dir`` key.
+
+Any number of other Toil options may also be specified. For defined Toil options,
+see the documentation:
+http://toil.readthedocs.io/en/latest/running/cliOptions.html
+
+Running WDL within Toil Scripts
```

Comment:
I think we should remove this section.  Users can run things with a Cromwell subprocess, but if we actually have decent WDL functionality, we should remove the recommendation of doing things that way.